### PR TITLE
Edited the code to blink LEDs on receipt of digits

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
changes done:
Created an header_file with two functions,one to blink the LEDs and the other to reset them on receipt of a non-digit input
Included the header file in main.c
Removed the delay() in main.c to do away with delays in updating the LEDs
Included an if statement in main.c to confirm for input of a digit
Added a loop to constantly poll the uart channel for data
Removed the function toogle_LED1 since its not needed

### CODE REVIEW

File: led_blinker.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+void led_blinker(char);
+void reset(void);
+void  led_blinker(char byte1){
+	  int i,k;
+	  int LED[4];//An array of four leds
+	  for (i = 0; i < 4; i++) {//looping through the byte 
+	  k = (( byte1 << i));	//left bitshifting
+
+	   if (k & 0x08){	//applying bitwise operator AND to k and the mask
```
Comment (Muriukidavid) [inline led_blinker.h]: its simpler to say

**Comment by Muriukidavid** [comment]
:-1: 
Don't use 
```c
#include<ctype.h>
```
write simpler code

**Comment by meshack-mbuvi** [comment]
I must score the 10/10 sir...you can't say nobody would...

**Comment by Muriukidavid** [comment]
I will post the total scores after you submit the third tutorial

On Thu, Mar 12, 2015 at 8:52 AM, meshack-mbuvi <notifications@github.com>
wrote:

> I must score the 10/10 sir...you can't say nobody would...
>
>
> On Wed, Mar 11, 2015 at 7:47 PM, David Muriuki <notifications@github.com>
> wrote:
>
> > Closed #5 <https://github.com/Muriukidavid/SPH336_Jan2015/pull/5>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/Muriukidavid/SPH336_Jan2015/pull/5#event-250248091>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Muriukidavid/SPH336_Jan2015/pull/5#issuecomment-78427724>
> .
>



-- 
-------------------------
Karibe
http://karibe.co.ke

**Pull request closed without merging**